cluster-autoscaler/simulator/framework/handle.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package framework
+
+import (
+	"context"
+	"fmt"
+
+	"k8s.io/client-go/informers"
+	"k8s.io/kubernetes/pkg/scheduler/apis/config"
```

Comment:
yikes having `"k8s.io/kubernetes/pkg/scheduler/apis/config"` and `"k8s.io/kubernetes/pkg/scheduler/apis/config/latest"` in the same import block

I think this is probably the best solution

```
scheduler_config "k8s.io/kubernetes/pkg/scheduler/apis/config"
scheduler_config_latest "k8s.io/kubernetes/pkg/scheduler/apis/config/latest"
```